go/vt/wrangler/fake_tablet_test.go

```diff
@@ -23,6 +23,9 @@ import (
 	"testing"
 	"time"
 
+	vdiff2 "vitess.io/vitess/go/vt/vttablet/tabletmanager/vdiff"
```

Comment:
Any reason for this alias?

Reply:
There is a type `vdiff` defined in `wrangler`